.github/workflows/ngc_reusable.yml

```diff
@@ -0,0 +1,66 @@
+name: NGC Reusable workflow
```

Comment:
confused about the NGC docker, can all PyTorch CI job use it as devel Docker?

Reply:
This NGC docker is prepared for the ONNX-TensorRT backend's benchmark.

The NGC docker is officially released by NVIDIA and keep updated monthly,  with the following libraries installed:
[CUDA](https://developer.nvidia.com/cuda-toolkit)
[cuBLAS](https://developer.nvidia.com/cublas)
[NVIDIA cuDNN](https://developer.nvidia.com/cudnn)
[NVIDIA NCCL](https://developer.nvidia.com/nccl) (optimized for [NVLink](https://www.nvidia.com/object/nvlink.html))
[RAPIDS](https://rapids.ai/)
[NVIDIA Data Loading Library (DALI)](https://developer.nvidia.com/dali)
[TensorRT](https://developer.nvidia.com/tensorrt)
[Torch-TensorRT](https://nvidia.github.io/Torch-TensorRT/)

Reply:
Thanks, as your introduction, I prefer to schedule the benchmark job daily or weekly, ref: https://github.com/marketplace/actions/schedule-job-action#define-scheduled-job

Reply:
Another but not related with this PR, how do we want to show the benchmark numbers? I found some pages with Google...
- https://labs.quansight.org/blog/2021/08/github-actions-benchmarks/
- https://github.com/marketplace/actions/continuous-benchmarking-for-go

It seems that we need a web server to vistualize these numbers...

Reply:
Done. I added a crontab.

Reply:
nit: If the file name and stage name could be renamed to benchmark-bla-bla-bla?